main.py

```diff
@@ -302,6 +307,16 @@ def create_circuit(self, request: Request) -> JSONResponse:
                    content=map_evc_event_content(evc))
         return JSONResponse(result, status_code=status)
 
+    def _use_uni_tags(self, evc):
+        uni_a = evc.uni_a
+        evc._use_uni_vlan(uni_a)
```

Comment:
Shouldn't this be also in the try catch scope?

Reply:
Added catching `ValueError` and re-raising error.